jpa/org.eclipse.persistence.jpa.jpql/src/test/java/org/eclipse/persistence/jpa/tests/jpql/parser/JPQLParserTest.java

```diff
@@ -3795,7 +3816,7 @@ public void test(Expression expression) {
             assertEquals(toString(),                       rangeVariableDeclaration.toParsedText());
             assertEquals(hasAs,                            rangeVariableDeclaration.hasAs());
             assertEquals(hasSpaceAfterAs,                  rangeVariableDeclaration.hasSpaceAfterAs());
-            assertEquals(!identificationVariable.isNull(), rangeVariableDeclaration.hasIdentificationVariable());
+//            assertEquals(!identificationVariable.isNull(), rangeVariableDeclaration.hasIdentificationVariable());
```

Comment:
Why this is commented out?

Reply:
Fixed. I forgot to uncomment after I commented it out when debugging.